app/lib/provider/persistence_provider.dart

```diff
@@ -112,6 +113,10 @@ class PersistenceService {
       await prefs.setInt(_version, 1);
     }
 
+    if (prefs.getInt(_anonymousAccess) == null) {
```

Comment:
You don't need to initialize it here.
Initialization is only needed if you don't have a stable default value but I have seen that you have a default value (line 242)

Reply:
it's removed